go-controller/pkg/ovn/obj_retry.go

```diff
@@ -868,22 +890,23 @@ func (oc *Controller) iterateRetryResources(r *retryObjs, updateAll bool) {
 		backoff := (entry.backoffSec * time.Second) + (time.Duration(rand.Intn(500)) * time.Millisecond)
 		objTimer := entry.timeStamp.Add(backoff)
 		if !updateAll && now.Before(objTimer) {
-			klog.V(5).Infof("%v retry %s not after timer yet, time: %s", r.oType, objKey, objTimer)
+			klog.V(5).Infof("Attempting retry of %s %s before timer (time: %s): skip", r.oType, objKey, objTimer)
 			continue
 		}
 
-		klog.Infof("%v %s: retry object setup", r.oType, objKey)
-
+		klog.Infof("Retry object setup: %s %s", r.oType, objKey)
 		// delete old object if needed
 		if entry.oldObj != nil {
-			klog.Infof("%v retry: removing old object for %s", r.oType, objKey)
+			klog.Infof("Removing old object: %s %s", r.oType, objKey)
 			if !isResourceScheduled(r.oType, entry.oldObj) {
 				klog.V(5).Infof("Retry: %s %s not scheduled", r.oType, objKey)
+				entry.failedRetries++
```

Comment:
why inc here we aren't scheduled for retry yet right ?

Reply:
Yeah, I wasn't 100% sure here. Can a pod be *never* scheduled? Should we keep retrying forever in that case? 
I'm lacking some hands-on experience on what happens with these pods that fail to be added/deleted...

@trozet any input? :)

Reply:
sure a pod can never be scheduled. If it has a node selector that isn't applicable (either the selector doesnt apply to any nodes, or the nodes it does apply to are unready. In either case once the pod is scheduled, we would get a pod update event. I think we can try a number of times and give up or just ignore pods that are not scheduled and not add them to retry. Either is fine.